src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/mobile/MobileAdsBaseObject.java

```diff
@@ -51,19 +53,25 @@ protected void setWindowSizeAndroid() {
 
     public void verifyMobileTopLeaderboard() {
         extractGptInfo(presentLeaderboardSelector);
-        removeSmartBanner();
-        if (checkIfElementOnPage(FLITE_MASK_SELECTOR)) {
-            PageObjectLogging.log(
-                    "FliteAd", "Page contains the flite ad", true, driver
-            );
-            return;
-        }
-        waitForSlotExpanded(presentLeaderboard);
+
+        removeElementIfPresent(SMART_BANNER_SELECTOR);
+
         if (!adsComparison.isAdVisible(presentLeaderboard, presentLeaderboardSelector, driver)) {
+
+            if (checkIfElementOnPage(CELTRA_MASK_SELECTOR)) {
+                PageObjectLogging.logWarning("Special ad", "Celtra");
```

Comment:
What do you think about logging also line-item-it and creative-id here? Would it be useful?